lib/reunion.rb

```diff
@@ -0,0 +1,74 @@
+require 'pry'
+require './lib/activity'
+
+class Reunion
+    attr_reader :name, :activities
+
+  def initialize(name)
+    @name = name
+    @activities = []
+  end
+
+  def add_activity(activity)
+    @activities << activity
+  end
+
+  def total_cost
+    sum = 0
+    @activities.each do |activity|
+      sum += activity.participants.values.sum
```

Comment:
Looks like this will work, but recommend using the `total_cost` method that you built on Activity.